comment.html

```diff
@@ -0,0 +1,93 @@
+<html>
+    <head>
+        <title>Comments (Have a suggestion?)</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+        <link href='https://fonts.googleapis.com/css?family=Lato:300,400,700' rel='stylesheet' type='text/css'>
+        <link href='custom.css' rel='stylesheet' type='text/css'>
+    </head>
+    <body>
+
+        <div class="container">
+
+            <div class="row">
+
+                <div class="col-lg-8 col-lg-offset-2">
+
+                    <h1>Help us improve smalldawn</a></h1>
+
+                    <p class="lead">Let us know if you have any issues finding a poem or how we can improve this site</p>
+
+
+                    <form id="contact-form" method="post" action="contact.php" role="form">
```

Comment:
form id="contact-form" **_method="post"_** **_action="contact.php"_** role="form"

So this part with the action = contact.php and the method = post  both are currently issue with the site because we do not have a back end and since we would most likely use a java script framework these php methods would not work out.